libs/backend/src/scan/cast_vote_records/export.ts

```diff
@@ -321,21 +318,54 @@ async function exportMetadataFileToUsbDrive(
       buildCastVoteRecordReportMetadata(exportContext),
     castVoteRecordRootHash,
   };
+  const metadataFileContents = JSON.stringify(metadata);
 
   const exportResult = await exporter.exportDataToUsbDrive(
     exportDirectoryPathRelativeToUsbMountPoint,
     'metadata.json',
-    JSON.stringify(metadata)
+    metadataFileContents
   );
   if (exportResult.isErr()) {
     return exportResult;
   }
 
-  // TODO: Sign metadata file
+  return ok({ metadataFileContents });
+}
+
+/**
+ * Exports a signature file for a cast vote record export
+ */
+async function exportSignatureFileToUsbDrive(
+  exportContext: ExportContext,
+  metadataFileContents: string,
+  exportDirectoryPathRelativeToUsbMountPoint: string
+): Promise<Result<void, ExportCastVoteRecordToUsbDriveError>> {
+  const { exporter } = exportContext;
+
+  const signatureFile = await prepareSignatureFile({
+    type: 'cast_vote_records',
+    context: 'export',
+    directoryName: path.basename(exportDirectoryPathRelativeToUsbMountPoint),
+    metadataFileContents,
+  });
+
+  const exportResult = await exporter.exportDataToUsbDrive(
+    // The signature file should live adjacent to the export directory, not within it.
+    path.parse(exportDirectoryPathRelativeToUsbMountPoint).dir,
+    signatureFile.fileName,
+    signatureFile.fileContents
+  );
+  if (exportResult.isErr()) {
+    return exportResult;
+  }
```

Comment:
I don't have a sense of how this would have been organized when the `auth` lib was writing the file, but it reads very nicely now.